stage/unstable/package-model.json

```diff
@@ -76,6 +76,10 @@
       "ref": "master",
       "source": "https://github.com/regolith-linux/libtrawldb.git"
     },
+    "meta-packages": {
+      "ref": "main",
+      "source": "https://github.com/regolith-linux/meta-packages.git"
```

Comment:
BTW, if we opt to add a debootstick-based release, it would be nice to have a metapackage that describes the set of packages for that as well.